.github/workflows/rust.yml

```diff
@@ -102,10 +102,10 @@ jobs:
             emulator-arch: x86_64
             # Note that x86_64 image is only available for API 21+. See
             # https://github.com/ReactiveCircus/android-emulator-runner?tab=readme-ov-file#configurations.
-            api-level: 21
+            api-level: 26
```

Comment:
I think this is conceptually unrelated and thus merits a separate commit.

Reply:
:+1: sorry for not doing that earlier. I updated the pull request @djc.